website/templates/project/register_draft.mako

```diff
@@ -25,14 +25,23 @@
     <div class="row-md-12 scripted">
         <span data-bind="ifnot: draft.isPendingApproval">
           <a type="button" class="btn btn-default pull-left" href="${draft['urls']['edit']}">Continue editing</a>
-
+          %if waffle.switch_is_active(features.OSF_PREREGISTRATION):
+          <button type="button" class="btn btn-success pull-right"
+                  data-toggle="tooltip" data-placement="top" title="The Prereg Challenge has expired"
+                  style="margin-left: 5px;
```

Comment:
Weird use of style tags here because the `disabled` class unjustly disables tooltips, so it's hacked to look and act disabled,